React/Views/RCTShadowView.h

```diff
@@ -8,6 +8,8 @@
 #import <UIKit/UIKit.h>
 
 #import <React/RCTComponent.h>
+// Keeps RCTConvert.h here before yoga for clang module to generate correct header imports.
+#import <React/RCTConvert.h>
```

Comment:
As far as I can see, `RCTConvert.h` also import `yoga/Yoga`, so we could also remove line 15, but I don't think it makes any harm to keep it.

Reply:
I can remove it, no problem. Should I?

Reply:
I would do it:
1. if it breaks on CI
2. only locally for knowledge's sake

So, probably no... :P